src/libAtomVM/term.h

```diff
@@ -88,6 +88,8 @@ extern "C" {
 #define CONS_SIZE 2
 #define REFC_BINARY_CONS_OFFSET 4
 #define LIST_SIZE(num_elements, element_size) ((num_elements) * ((element_size) + CONS_SIZE))
+#define TERM_MAP_SIZE(num_elements) (3 + 2 * num_elements)
```

Comment:
`(3 + 2 * num_elements)` -> `(3 + 2 * (num_elements))`
Otherwise doing `TERM_MAP_SIZE(count_a + count_b)` will be expacted to something quite unexpected.